### ISSUE TITLE
Super sized song cover jpg

### ISSUE THREAD

**Issue by Gamegod4** [andruzzzhka/BeatSaberMultiplayer#115]
Something that seems to be happening when playing multiplayer is that sometimes the room is bugged in an odd way. There's a huge jpg of the kda song cover thats like 3-4 times the size of an avatar that is just standing in the center of the room. This wouldn't be a huge issue, if it didn't also seem to cause the menu buttons to not work. If not hosting it doesn't really do anything to you, but I didn't host any rooms recently. I thought I would bring it up since I've seen a streamer's playback also had it happen so it may not be something just on my side. Restarting the game also fixes it pretty much all the time. Might be caused from coming from solo play

**Comment by sehyeon0** [comment]
I figured I would also try to add to this by posting a video of what actually is shown. Played some multiplayer last night and it had appeared after coming from solo play to multiplayer. After a game restart and going straight into multiplayer it has vanished. I was not the host of the game and eventually out of the 7 players we had only 2 of us had seen it. 

I don't have contact with the other person who had seen it so I'm not sure if they came from solo play or not but I hope being able to see what happens will help in some way. If I can provide any other info that might be helpful (like mods I have installed? idk) let me know. 

thanks for working on this mod ^^ <3 

video: https://streamable.com/ylqi6
